pyproject.toml

```diff
@@ -82,11 +78,22 @@ warn_return_any = true
 warn_unused_ignores = true
 show_error_codes = true
 
-[tool.flake8]
-max-line-length = 88
+[tool.ruff]
+line-length = 88
+target-version = "py310"
+show-fixes = true
+exclude = ["**/migrations"]
+
+[tool.ruff.lint]
+select = ["A", "B", "C", "E", "F", "G", "I", "N", "Q", "S", "W", "PL", "UP", "RUF"]
```

Comment:
If you're adding more options to the standard config, consider using select-extend instead of select